lib/pages/home/home_drawer_widget.dart

```diff
@@ -303,6 +304,9 @@ class HomeDrawerWidget extends StatelessWidget {
     DefaultTabController.of(context)?.animateTo(0);
 
     Get.find<CompaniesCubit>().selectWorkspace(workspaceId: workspaceId);
+
+    AccountRepository().setRecentWorkspace();
```

Comment:
Should not call directly `AccountRepository` here. Our project is following bloc pattern, `data` and `UI` should communicate via `cubit` layer (`AccountCubit`)